src/App.js

```diff
@@ -1,23 +1,20 @@
-import React, { Component } from 'react';
+import React from 'react';
 import Header from './component/Header';
 import Work from './component/Work';
 import Project from './component/Project';
 import Education from './component/Education';
 import Footer from './component/Footer';
+import Data from './data'
 import './App.css';
 
-class App extends Component {
-  render() {
-    return (
-      <div>
-        <Header />
-        <Work />
-        <Project />
-        <Education />
-        <Footer />
-      </div>
-    );
-  }
-}
+const App = (props) => (
```

Comment:
This could be reverted back to Component, and load the data via constructor

```
class App extends Component {
  constructor(props){
    super(props);
    this.state = {data: Data}
  }

  render() {
     const { data } = this.state;
     // Rendering codes...
     // ...
}
```